src/main/java/mekanism/tools/common/ToolsItems.java

```diff
@@ -1,295 +0,0 @@
-package mekanism.tools.common;
-
-import mekanism.tools.item.ItemMekanismArmor;
-import mekanism.tools.item.ItemMekanismAxe;
-import mekanism.tools.item.ItemMekanismHoe;
-import mekanism.tools.item.ItemMekanismPaxel;
-import mekanism.tools.item.ItemMekanismPickaxe;
-import mekanism.tools.item.ItemMekanismShovel;
-import mekanism.tools.item.ItemMekanismSword;
-import net.minecraft.inventory.EntityEquipmentSlot;
-import net.minecraft.item.Item;
-import net.minecraft.item.Item.ToolMaterial;
-import net.minecraft.util.ResourceLocation;
-import net.minecraftforge.fml.common.registry.GameRegistry.ObjectHolder;
-import net.minecraftforge.registries.IForgeRegistry;
-
-@ObjectHolder(MekanismTools.MODID)
```

Comment:
As much as I hate `ObjectHolder`, the armour is probably the one case it _should_ be used, on the off chance that someone does a registry replacement on us.

Reply:
If someone really wants to do a registry replacement on us, I believe they still can with reflection and it is a lot cleaner to just be able to loop over everything. (If you want I could add a method that makes it so they don't *have* to use reflection to change the enum's `Item` reference.

Reply:
The fact is with object holders they dont have to do _anything_ special